src/views/Statistic/Statistic.jsx

```diff
@@ -0,0 +1,42 @@
+import { classes, getChildProps, getElementType } from '../../lib';
+import { Enum } from '../../lib/PropTypes';
+
+export default {
+  name: 'SuiStatistic',
+  props: {
+    horizontal: {
+      type: Boolean,
+      description: 'Present measurement horizontally',
+    },
+    color: Enum.Color,
+    size: Enum.Size,
+    floated: Enum(['left', 'right']),
+    inverted: {
+      type: Boolean,
+      description: 'Should the colors be inverted',
+    },
+    inGroup: {
+      type: Boolean,
+      description: 'For now manual override, when used inside Statistic Group, to avoid adding extra ui class',
+    },
```

Comment:
I get what you wanted to achieve here, but the additional `ui` class would not break anything.
In fact, in your examples you're using camel case for props (it should be `in-group`, not `inGroup`) so you aren't using this one at all. I'd say to remove it completely

Reply:
For this particular example - it seems to me, that it does have an affect. 
Pics for proof of how blocks behave with or without `.ui > .ui`:
https://i.imgur.com/AmlqDpo.png
https://i.imgur.com/4rQ78Mx.png

Should I still remove the inGroup logic?

As for camelCase - I think vue magically does some converting under the hood, so camelCase and `in-group` will both work:
https://i.imgur.com/sVX07CV.png

Reply:
You're right, it is making a small difference. Let's leave it like this for now, I'll come up with a solution later on.

Reply:
Great! Thanks!